2025-05-18 21:57:25,990 INFO    Thread-13 (_run_job):967128 [wandb_init.py:setup_run_log_directory():662] Logging user logs to /speech/utkarsh/da6401_assignment3/wandb/run-20250518_215725-oep7bkg2/logs/debug.log
2025-05-18 21:57:25,990 INFO    Thread-13 (_run_job):967128 [wandb_init.py:setup_run_log_directory():663] Logging internal logs to /speech/utkarsh/da6401_assignment3/wandb/run-20250518_215725-oep7bkg2/logs/debug-internal.log
2025-05-18 21:57:25,990 INFO    Thread-13 (_run_job):967128 [wandb_init.py:init():781] calling init triggers
2025-05-18 21:57:25,990 INFO    Thread-13 (_run_job):967128 [wandb_init.py:init():786] wandb.init called with sweep_config: {'batch_size': 128, 'beam_size': 7, 'cell_type': 'lstm', 'decoder_dropout': 0.5, 'embedding_size': 256, 'encoder_dropout': 0.5, 'hidden_size': 64, 'learning_rate': 0.0005, 'num_layers': 2, 'use_attention': False}
config: {'data_path': 'dakshina_dataset_v1.0/hi/lexicons/', 'embedding_size': 256, 'hidden_size': 64, 'num_encoder_layers': 2, 'num_decoder_layers': 2, 'encoder_dropout': 0.5, 'decoder_dropout': 0.5, 'cell_type': 'lstm', 'batch_size': 128, 'learning_rate': 0.0005, 'n_epochs': 20, 'clip': 1.0, 'teacher_forcing_ratio': 0.5, 'patience': 5, 'min_delta': 0.0, 'beam_size': 7, 'save_path': 'models/', 'model_name': 'best_model.pt', 'seed': 42, 'log_wandb': True, 'wandb_project': 'da6401_assignment3', 'wandb_name': 'lstm-ec_2-dc_2-hs_64-emb_256-bs_128-lr_0.0005', 'use_attention': False, '_wandb': {}}
2025-05-18 21:57:25,990 INFO    Thread-13 (_run_job):967128 [wandb_init.py:init():801] wandb.init() called while a run is active
2025-05-18 22:15:47,405 INFO    Thread-13 (_run_job):967128 [wandb_run.py:_finish():2189] finishing run da24s011-indian-institute-of-technology-madras/da6401_assignment3/oep7bkg2
2025-05-18 22:15:47,405 INFO    Thread-13 (_run_job):967128 [wandb_run.py:_atexit_cleanup():2419] got exitcode: 1
2025-05-18 22:15:47,405 INFO    Thread-13 (_run_job):967128 [wandb_run.py:_restore():2401] restore
2025-05-18 22:15:47,405 INFO    Thread-13 (_run_job):967128 [wandb_run.py:_restore():2407] restore done
2025-05-18 22:15:48,899 INFO    Thread-13 (_run_job):967128 [wandb_run.py:_footer_history_summary_info():4064] rendering history
2025-05-18 22:15:48,899 INFO    Thread-13 (_run_job):967128 [wandb_run.py:_footer_history_summary_info():4096] rendering summary
2025-05-18 22:15:48,900 INFO    Thread-13 (_run_job):967128 [wandb_run.py:_footer_sync_info():4025] logging synced files
2025-05-18 22:15:48,904 ERROR   MainThread:967128 [pyagent.py:_run_jobs_from_queue():234] [no run ID] Run oep7bkg2 errored:
Traceback (most recent call last):
  File "/speech/utkarsh/.local/lib/python3.10/site-packages/wandb/agents/pyagent.py", line 306, in _run_job
    self._function()
  File "/speech/utkarsh/da6401_assignment3/sweep.py", line 50, in sweep_train
    main(model_config)
  File "/speech/utkarsh/da6401_assignment3/main.py", line 124, in main
    examples = visualize_attention(
  File "/speech/utkarsh/da6401_assignment3/utils.py", line 177, in visualize_attention
    output_indices, attention_weights = model.greedy_decode(src_tensor, max_len=100)
ValueError: too many values to unpack (expected 2)

2025-05-18 22:15:51,901 INFO    MsgRouterThr:967128 [mailbox.py:close():129] [no run ID] Closing mailbox, abandoning 1 handles.
2025-05-18 22:21:57,476 ERROR   MainThread:967128 [pyagent.py:_run_jobs_from_queue():234] [no run ID] Run zdm5nhi0 errored:
Traceback (most recent call last):
  File "/speech/utkarsh/.local/lib/python3.10/site-packages/wandb/agents/pyagent.py", line 306, in _run_job
    self._function()
  File "/speech/utkarsh/da6401_assignment3/sweep.py", line 50, in sweep_train
    main(model_config)
  File "/speech/utkarsh/da6401_assignment3/main.py", line 124, in main
    examples = visualize_attention(
  File "/speech/utkarsh/da6401_assignment3/utils.py", line 177, in visualize_attention
    output_indices, attention_weights = model.greedy_decode(src_tensor, max_len=100)
ValueError: too many values to unpack (expected 2)

2025-05-18 22:22:01,475 INFO    MsgRouterThr:967128 [mailbox.py:close():129] [no run ID] Closing mailbox, abandoning 1 handles.
2025-05-18 22:39:08,364 ERROR   MainThread:967128 [pyagent.py:_run_jobs_from_queue():234] [no run ID] Run 9gdenox8 errored:
Traceback (most recent call last):
  File "/speech/utkarsh/.local/lib/python3.10/site-packages/wandb/agents/pyagent.py", line 306, in _run_job
    self._function()
  File "/speech/utkarsh/da6401_assignment3/sweep.py", line 50, in sweep_train
    main(model_config)
  File "/speech/utkarsh/da6401_assignment3/main.py", line 124, in main
    examples = visualize_attention(
  File "/speech/utkarsh/da6401_assignment3/utils.py", line 177, in visualize_attention
    output_indices, attention_weights = model.greedy_decode(src_tensor, max_len=100)
ValueError: too many values to unpack (expected 2)

2025-05-18 22:39:08,366 ERROR   MainThread:967128 [pyagent.py:_run_jobs_from_queue():259] [no run ID] Detected 5 failed runs in a row at start, killing sweep.
2025-05-18 22:39:09,358 INFO    MsgRouterThr:967128 [mailbox.py:close():129] [no run ID] Closing mailbox, abandoning 1 handles.
